src/go/rpk/pkg/cli/cmd/group/seek.go

```diff
@@ -131,7 +131,7 @@ func parseSeekFile(
 ) (kadm.Offsets, error) {
 	f, err := fs.Open(file)
 	if err != nil {
-		return nil, fmt.Errorf("unable to open %q: %v", file, err)
+		return nil, fmt.Errorf("Unable to open %q: %v", file, err)
```

Comment:
I'm not sure about capitalizing these. They're returned errors, and may be further wrapped before being printed. In Go, it's idiomatic to not capitalize errors. I've slowly been adding lowercase error text, and I think the lowercasing of errors is a clear "things went wrong" type of signal as well.

Reply:
Yeah I wasn't so sure about this one either. I'll return to the lowercase format.